music21/key.py

```diff
@@ -68,13 +68,26 @@ def convertKeyStringToMusic21KeyString(textString):
     'b#'
     >>> key.convertKeyStringToMusic21KeyString('c')
     'c'
+    >>> key.convertKeyStringToMusic21KeyString('Bbb')
+    'B--'
+    >>> key.convertKeyStringToMusic21KeyString('bbb')
+    'b--'
```

Comment:
Also a test for a different double-flat key than b, say 'Ebb' since that follows a completely different code path potentially than 'bbb'

Reply:
I think both cases follow the same path since the checks for B/b/Bb/bb only apply when the key has two or fewer characters. But I added such a test.

I wasn't sure what the right balance between comprehensiveness and readability was here since these are actually doctests (albeit ones that people who aren't actually developing music21 are probably unlikely to consult).